src/commands/prepare-commit-msg-hook.ts

```diff
@@ -34,12 +34,18 @@ export default () => (async () => {
 
 	const s = spinner();
 	s.start('The AI is analyzing your changes');
-	const messages = await generateCommitMessage(
-		config.OPENAI_KEY,
-		config.locale,
-		staged!.diff,
-		config.generate,
-	);
+	let messages;
+	try {
+		messages = await generateCommitMessage(
+			config.OPENAI_KEY,
+			config.locale,
+			staged!.diff,
+			config.generate,
+		);
+	} catch (error) {
+		s.stop('The AI is analyzing your changes');
+		throw error;
+	}
```

Comment:
Change to `finally` block so we don't need to catch or re-throw the error.

Reply:
I changed it to finally block and keep the stop message the same in both cases.